graders/r/README.md

```diff
@@ -1,10 +1,31 @@
-## PrairieLearn Autograding R Docker Image
+## rocker-pl
 
+This is a Docker image working as an external grader to be paired with PrairieLearn.  It has been
+used in the [STAT 430](https://stat430.com) (2018-2020) and [STAT 447](https://stat430.com) (since
+2021) courses on _Data Science Programming Methods_ at the University of Illinois at
+Urbana-Champaign. In a previous course iteration, a CentOS7-based container was used via Python. We
+now drive R directly from PrairieLearn evaluating code in this container. However, the container is
+perfectly generic and can be used for general R work as well.
```

Comment:
I suggest leaving out historical information unless it's important for understanding the current code?
```suggestion
Urbana-Champaign. However, the container is
perfectly generic and can be used for general R work as well.
```